Model/Config.php

```diff
@@ -313,9 +315,8 @@ public function isActivated($scopeId = null, $scope = null)
      */
     public function setStoreCredentialsAndIsEnabled($appKey, $secret, $isEnabled, $storeId = null, $scopes = ScopeInterface::SCOPE_STORES)
     {
-        if (is_null($storeId)) {
-            $storeId = $this->storeManager->getStore()->getId();
-        }
+        $storeId = ($storeId !== null) ?: $this->storeManager->getStore()->getId();
```

Comment:
This is incorrect, should stay unchanged or be:
`$storeId = ($storeId === null) ? $this->storeManager->getStore()->getId(); : $storeId`

Reply:
Fixed, with small addition - I use $this->getCurrentStoreId().